Gaming The Syllabus/Assets/Scripts/TooltipManager.cs

```diff
@@ -0,0 +1,48 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.InputSystem;
+using TMPro;
+
+public class TooltipManager : MonoBehaviour
+{
+    public static TooltipManager _instance;
+
+    public TextMeshProUGUI textComponent;
+
+    void Awake()
+    {
+        if (_instance != null && _instance != this)
+        {
+            Destroy(this.gameObject);
+        }
+        else
+        {
+            _instance = this;
+        }
+    }
+    // Start is called before the first frame update
+    void Start()
+    {
+        Cursor.visible = true;
```

Comment:
Wonder why we need this here :0
Was the cursor invisible otherwise? If so maybe we should put this somewhere more central, seems easy to forget where this is but I also can't immediately think of a better place so idk